### PR TITLE
fix: Configurable backoff strategy for DBConnection

### PR DESCRIPTION
**Pull request by filipecabaco**
## What kind of change does this PR introduce?

Configurable backoff strategy for DBConnection

### CODE REVIEW

File: lib/realtime/helpers.ex [hunk 1]
```diff
@@ -87,7 +92,8 @@ defmodule Realtime.Helpers do
         pool \\ 5,
         queue_target \\ 5_000,
         ssl_enforced \\ true,
-        application_name \\ "realtime_supabase"
+        application_name \\ "realtime_supabase",
+        backoff_type \\ :exp
```
Comment (chasers) [inline lib/realtime/helpers.ex:96]: Default for db connection is rand_exp right? Should we stick with that for the default?

**Comment by chasers** [comment]
@filipecabaco don't you want to use `:stop` when calling `connect_db` from the `Connect` module?

**Comment by filipecabaco** [comment]
that is done with the `check_tenant_connection`. at the moment that is the only time I'm connecting

File: lib/realtime/helpers.ex (hunk 1 above)
Comment (filipecabaco) [reply lib/realtime/helpers.ex:96]: will change

**Review by chasers — APPROVED** [review]

**Comment by chasers** [comment]
> that is done with the `check_tenant_connection`. at the moment that is the only time I'm connecting

Ahh missed that. LGTM!

**Pull request merged**